intro_to_rl.ipynb

```diff
@@ -0,0 +1,3225 @@
+{
```

Comment:
<p>Nit: there is a lot of inconsistent spacing with function args. Either a space or no space after comma.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>We should run linters through all pracs to avoid this. </p>

Reply:
Yeah good idea. Tried to improve things where I could.